src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={`TodoInfo ${todo.completed
+      ? 'TodoInfo--completed'
+      : ''}`}
+  >
```

Comment:
I think the classNames library would look good here

Reply:
Good point. [Read more here](https://www.npmjs.com/package/classnames)